tests/migration/online_migration/pre_migration.pm

```diff
@@ -48,6 +48,9 @@ sub remove_kgraft_patch {
 sub run {
     select_console 'root-console';
 
+    # set_zypp_single_rpmtrans
+    # if libzypp-17.28.4 and zypper-1.14.49 or higher version installed
+    set_zypp_single_rpmtrans();
```

Comment:
Maybe we need setting for control this behavior?

Reply:
I think so, with a setting to control this can be more flexibly.

Reply:
yeah, compromise is OK for me.